src/modules/search/search.js

```diff
@@ -1,6 +1,13 @@
-goog.provide('ngeo.searchDirective');
+/**
+ * @module ngeo search namespace
+ */
+goog.provide('ngeo.modules.search');
```

Comment:
In the prospect of migrating one day to ES6 modules I kept the naming of the filesystem.